dali/test/python/test_operator_nonsilence.py

```diff
@@ -35,6 +35,8 @@ def trim_ref(top_db, ref, frame_length, hop_length, input_data):
     # while DALI uses beginning of window. Hence the subtraction below
     begin = index[0] - frame_length / 2
     length = index[1] - index[0]
+    if length != 0:
```

Comment:
I think we should be giving the end of the window as the end of the non silence range (as I described above)